spec/rubocop/cop/rspec/implicit_subject_spec.rb

```diff
@@ -98,6 +107,17 @@ def permits(actions)
       RUBY
     end
 
+    it 'allows `is_expected` in single statement with an alias' do
```

Comment:
The root problem was that Includes.examples were not considered, so maybe it makes sense to specify this, e.g. "with an alias to Includes.examples".

On a different thought, `its_call` doesn't seems to be a very fair alias to e.g. `it_behaves_like`, as the latter creates an example group.
Perhaps it would make sense to change aliasing in `saharspec` to `RSpec/Language/Examples/Regular` [just like we have it for `its`](https://github.com/rubocop/rubocop-rspec/blob/bb046ebb8ede99ea9273240918b6b22a516a3dbe/config/default.yml#L48).
This would also make this PR redundant, and would also 

See https://github.com/palkan/action_policy/pull/138 for more, but there is some specifics to how e.g. `success` is used there, without a block to justify adding it as an alias to Include.examples.

What do you think?

Reply:
@pirj do you ask if the following config is equal to the one that I made in the PR to saharspec?

```yml
RSpec:
  Language:
    Examples:
      Regular:
        - its_block
        - its_call
        - its_map
      Skipped:
        - xits_block
        - xits_call
        - xits_map
      Focused:
        - fits_block
        - fits_call
        - fits_map
```

I think so, I checked this config and it works well for all my scenarios.